src/riak_kv_entropy_manager.erl

```diff
@@ -189,6 +208,15 @@ expire_trees() ->
 clear_trees() ->
     gen_server:cast(?MODULE, clear_trees).
 
+get_version() ->
```

Comment:
Yeah, I'll add it to all these api calls. You're right, public facing API calls should absolutely have specs.